change/@ni-nimble-angular-862e5fd9-2cd6-456a-b26b-eaf3477cbd83.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
As long as the table is considered "experimental" we can keep the change type as "patch" to signal that this isn't adding any new APIs that clients should actually use. Same for all the packages.

Reply:
Updated.